package.json

```diff
@@ -23,7 +23,7 @@
     "!src/rules/**/*.test.js"
   ],
   "engines": {
-    "node": ">=8.0.0"
```

Comment:
This is not a breaking change because we already have many dependencies that only support node 12+